jobs/configure-eirini/templates/run.erb

```diff
@@ -1,27 +1,22 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-status "Configuring Eirini"
```

Comment:
We didn't have the `status` command in our stemcell. We assumed it was ok to replace with echo, but not sure if we're removing some functionality by doing son.

Reply:
https://github.com/cloudfoundry-community/eirini-bosh-release/pull/25/files#diff-1ff2454ad97a02bc44853aa5dcde2a90R3